src/meltano/api/controllers/plugins.py

```diff
@@ -147,12 +147,12 @@ def install_batch():  # noqa: WPS210
     )
 
     add_service = ProjectAddService(project, plugins_service=plugins_service)
-    related_plugins = add_service.add_related(plugin)
+    required_plugins = add_service.add_required(plugin)
 
     # We will install the plugins in reverse order, since dependencies
-    # are listed after their dependents in `related_plugins`, but should
+    # are listed after their dependents in `required_plugins`, but should
     # be installed first.
-    related_plugins.reverse()
+    required_plugins.reverse()
```

Comment:
I *think* this can be omitted when using add_required. Can you double check if you still actually need to reverse the list?

/cc @edgarrmondragon

Reply:
@pandemicsyn yeah, I think it should be safe to omit it

Reply:
I think I did something wrong when I merged the change.  😞